neo/rawio/spikeglxrawio.py

```diff
@@ -488,13 +513,15 @@ def extract_stream_info(meta_file, meta):
     else:
         # NIDQ case
         has_sync_trace = False
-    fname = Path(meta_file).stem
+
+    bin_file_path = meta["fileName"]
```

Comment:
This is the part that uses the meta["fileName"] (the original one) instead of the file_path in the system (which the user might have changed).